approval_frame/settings.py

```diff
@@ -9,7 +9,10 @@
 SECRET_KEY = 'jndet@)ed9ge4+!k605hc+$0tjb8u5oe)pdjab)-b3lsot_xnf'
```

Comment:
Yup that sounds good (I didn't know we were generating it till you pointed it out :smile_cat:)